### PR TITLE
Added link to Rubyinstaller in reference to installer for Windows

### PR DESCRIPTION
**Pull request by mandyreal**

### CODE REVIEW

File: src/pages/ruby/ruby-introduction/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The current stable version is 2.4.2\. It contains security improvements from its
 Mac OS X and many Linux distributions come pre-installed with Ruby. To check if ruby is pre-installed in your system, just run `ruby -v` on your shell. There are several ways to install Ruby:
 
 *   When you are on a UNIX-like operating system, using your system’s package manager is the easiest way of getting started. However, the packaged Ruby version usually is not the newest one.
-*   Installers can be used to install a specific or multiple Ruby versions. There is also an installer for Windows.
+*   Installers can be used to install a specific or multiple Ruby versions. There is also an installer for Windows using <a href='https://rubyinstaller.org' target='_blank' rel='nofollow'>RubyInstaller</a>.
```
Comment (jp-sauve) [inline src/pages/ruby/ruby-introduction/index.md]: Links should be in markdown, not html, per the style guide.
https://github.com/freeCodeCamp/guides#article-style-guide

Comment (mandyreal) [reply src/pages/ruby/ruby-introduction/index.md]: @jp-sauve Thanks for the feedback. The latest index.md links to all the available installers in the ruby-lang.org site which makes this change not relevant anymore. Let me know if the all links need to be changed to markdown, they're in html mostly it seems.

Comment (jp-sauve) [reply src/pages/ruby/ruby-introduction/index.md]: It's the original link submissions that should be in markdown, to make sure that the links get the correct attributes set. I believe the script adds rel='nofollow' and maybe a target. If those attributes aren't there, feel free to switch them to markdown!

**Comment by johnkennedy9147** [comment]
PR authors comment suggests he doesnt intend to make any changes as the article has already been updated by another PR. closing this PR

**Pull request closed without merging**